packages/lesswrong/components/form-components/ImageUploadDefaultsDialog.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+
+const styles = (theme: ThemeType): JssStyles => ({
+  images: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'space-evenly'
+  },
+  image: {
+    padding: '5px 5px 2px',
+    margin: 3,
+    border: '2px solid transparent',
+    cursor: 'pointer',
+    '&:hover': {
+      borderColor: theme.palette.primary.main,
+    },
+  },
+})
+
+const ImageUploadDefaultsDialog = ({ onSelect, onClose, classes }: {
+  onSelect: Function,
+  onClose: ()=>void,
+  classes: ClassesType,
+}) => {
+
+  const { LWDialog, CloudinaryImage } = Components
+  
+  const selectImg = (img) => {
+    onSelect(img)
+    onClose()
+  }
+  
+  // these are EA Forum-specific
+  const defaultImages = [
+    'Event/defaults/ucbrkw7gdmacm4soorui', // clouds
+    'Event/defaults/k7bdilxm08silijqdn2v', // sprout
+    
+    'Event/defaults/kaib61wcuwtsgmq6elrs', // autumn trail
+    'Event/defaults/hlmllj77ln4nr69fn7jf', // birds
+    
+    'Event/defaults/astdsmglxtbcehnlo7g4', // group hike
+    'Event/defaults/zhiyjstbfu1olxfbnuci', // group picnic
+    
+    'Event/defaults/fg3ttihepxfss9ylun4f', // dog on hill
+    'Event/defaults/dqknjubfjt2crsmdspsw', // veggies
+  
+    'Event/defaults/gtst9i5zvxk9hulnwigm', // books & coffee
+    'Event/defaults/as81227r6jx8m4hydhb3', // bookshelf
+    
+    'Event/defaults/qdfqhekgmxtf2wbdbrq3', // desk with laptop & notebook
+    'Event/defaults/y1bt55znqoc2lvec70wp', // code
+    
+    'Event/defaults/qvujytx4nomsrm3xu7on', // two people working
+    'Event/defaults/myksyxm0e1qszlbzzj9o', // post-its on wall
+  
+    'Event/defaults/fsgbi942lcnepj7zylr5', // coffee hangout
+    'Event/defaults/dgjptfxhtomlt5s96wgn', // wine hangout
+  
+    'Event/defaults/akwwm4hecuwypw5ixte0', // outdoor party
+    'Event/defaults/b8xpbqpegclnvc8x0taw', // sparkler
+    
+    'Event/defaults/cq99vatplysmhlv6drai', // virtual meeting
+    'Event/defaults/fs5rgzbtdnbcdegiszbw', // butterfly
+    
+    'Event/defaults/h4jbx7exu0ttrcopmvcy', // elephants
+    'Event/defaults/ycj85pqkcvp8q91rqn87', // sheep
+    
+    'Event/defaults/np7kecliyzhnlogyxelp', // pugsworth
+    'Event/defaults/diddy0dxe7lfoxnwtlvq', // jellyfish
+  
+    'Event/defaults/uqi9pobdxb9dxbpomep0', // cells
+    'Event/defaults/mtkocedjg5zx99ghkxnw', // pipette
+  
+    'Event/defaults/s4pkf3kuhnigiov8effy', // coins
+    'Event/defaults/ijfeayyzhxixbrxzvmm5', // lightbulb
+    
+    'Event/defaults/dic1sxm86xcvoavnhpnn', // wind farm & field
+    'Event/defaults/gumqg9zryaqirtldgast', // lake & mountains
+  
+    'Event/defaults/ubobbqry5uowozmptjop', // earth
+    'Event/defaults/n33epog75uk7c1dgciod', // starry night
+  ]
```

Comment:
I have a feeling a lot of these won't be used, and they're missing some easy ones that would be very useful, like pictures of people in EA shirts talking about things. I guess this updates me towards having these be easy to change. Can we make it so we get a list from a cloudinary folder so all we have to do is add to that folder to update the list.

Seems fine to do that in a follow-on PR / only if you get that feedback from a real user.

Reply:
What other images do you think are missing?

Reply:
@jpaddison3 I like your suggestions! I'm trying to avoid including people's faces in the default images, and I couldn't find a good photo of people in EA shirts talking that would feel anonymous enough to me. If you have suggestions or more specific thoughts on the default photos I'd love to hear them, especially what you think is missing. :)

It looks like we can get [all the images in a folder](https://support.cloudinary.com/hc/en-us/articles/202521082-How-to-list-all-images-within-a-folder-) from Cloudinary, but I will probably just release what I have here first. Partly because...

I agree that many of the defaults won't be used - of the EA events I've looked at on Facebook, surprisingly the vast majority have custom images, so I would guess most events will have custom-made images. I'm just indecisive so I ended up with a lot more defaults that I had anticipated. :)

I think if we get feedback that leads us to wanting to change the default images, or when LW wants to add their own default images, then it would be a good time for us to start pulling them from the folder directly.

Reply:
Maybe we can specifically get EA's to agree to model for the default images? Otherwise, reasonable counterarguments.